main/kotlin/com/example/people_managment_system/PostgresqlApplication.kt

```diff
@@ -0,0 +1,12 @@
+package com.example.people_managment_system
+
+import org.springframework.boot.autoconfigure.SpringBootApplication
+import org.springframework.boot.runApplication
+
+@SpringBootApplication
+class PostgreSqlApplication
+
+fun main(args: Array<String>) {
+	val runApplication = runApplication<PostgreSqlApplication>(*args)
```

Comment:
I think you can have it like: `runApplication<PostgreSqlApplication>(*args)` in which should also work.